src/nodes/math/OperatorNode.js

```diff
@@ -274,7 +316,20 @@ class OperatorNode extends TempNode {
 
 			} else {
 
-				return builder.format( `( ${ a } ${ op } ${ b } )`, type, output );
+				// Handle matrix operations
+				if ( builder.isMatrix( typeA ) && typeB === 'float' ) {
+
+					return builder.format( `(${b} ${op} ${a})`, type, output );
+
+				} else if ( typeA === 'float' && builder.isMatrix( typeB ) ) {
+
+					return builder.format( `(${a} ${op} ${b})`, type, output );
+
+				} else {
+
+					return builder.format( `(${a} ${op} ${b})`, type, output );
```

Comment:
As DeepScan noticed, the code in this branch and the previous one is identical.

Reply:
Got it! Fixed.